capa/rules/cache.py

```diff
@@ -102,30 +103,29 @@ def compute_ruleset_cache_identifier(ruleset: capa.rules.RuleSet) -> CacheIdenti
     return compute_cache_identifier(rule_contents)
 
 
-def cache_ruleset(cache_dir: str, ruleset: capa.rules.RuleSet):
+def cache_ruleset(cache_dir: Path, ruleset: capa.rules.RuleSet):
     """
-    cache the given ruleset to disk, using the given cache directory.
-    this can subsequently be reloaded via `load_cached_ruleset`,
+    Cache the given ruleset to disk, using the given cache directory.
+    This can subsequently be reloaded via `load_cached_ruleset`,
     assuming the capa version and rule content does not change.
 
-    callers should use this function to avoid the performance overhead
+    Callers should use this function to avoid the performance overhead
     of validating rules on each run.
     """
     id = compute_ruleset_cache_identifier(ruleset)
     path = get_cache_path(cache_dir, id)
-    if os.path.exists(path):
-        logger.debug("rule set already cached to %s", path)
+    if path.exists():
+        logger.debug("Rule set already cached to %s", path)
```

Comment:
please revert capitalization changes. we can consider updating the grammer in another PR. in the meantime, we want to avoid inconsistency in the codebase, even if the grammar is bad.